### PR TITLE
Blazor Pre8 Updates

### PR DESCRIPTION
**Pull request by guardrex**
This should take care of the remaining items.

* Not sure on adding additional info from *Improved reconnection logic for Blazor Server apps* in the blog post. Let me know.
* Check me on the VS UI steps for creating a new Razor Class Library in the *Class Libraries* topic.

I'll shoot this in after any of you have a chance to look it over.

### CODE REVIEW

**Comment by guardrex** [comment]
@danroth27 @rynowak @SteveSandersonMS I'm anxious to get this in asap.

I'm about to get an :ear:-ful of 💀 *death threats* 💀 from our readers. :smile:

**Comment by guardrex** [comment]
@ajaybhargavb Are you available to run a quick :eye: over this?

File: aspnetcore/blazor/components.md [hunk 1]
```diff
@@ -513,25 +513,27 @@ Prefer the strongly typed `EventCallback<T>` over `EventCallback`. `EventCallbac
 
 ## Capture references to components
 
-Component references provide a way to reference a component instance so that you can issue commands to that instance, such as `Show` or `Reset`. To capture a component reference, add a [@ref](xref:mvc/views/razor#ref) attribute to the child component and then define a field with the same name and the same type as the child component.
+Component references provide a way to reference a component instance so that you can issue commands to that instance, such as `Show` or `Reset`.
+
+The Razor compiler automatically generates a backing field for element and component references when using [@ref](xref:mvc/views/razor#ref). In the following example, there's no need to create a `myLoginDialog` field for the `LoginDialog` component:
```
Comment (ajaybhargavb) [inline aspnetcore/blazor/components.md]: @guardrex, we're reverting automatic backing field generation. @NTaylorMullen is there an issue filed for doing that?

Comment (guardrex) [reply aspnetcore/blazor/components.md]: Interesting ... I just spoke with Ryan offline yesterday about it, and he didn't mention it. Also, Dan's blog post goes this way. I *definitely* need guidance on how to handle it here.

**Comment by guardrex** [comment]
Thanks @ajaybhargavb ... I'll hold off until I hear more on the backing store. I'm just anxious to get this in because the docs are in a bit of a broken state right now vis-a-vis the Pre8 updates.

File: aspnetcore/blazor/components.md (hunk 1 above)
Comment (rynowak) [reply aspnetcore/blazor/components.md]: Yes @ajaybhargavb is correct. Unfortunately there are **fundamental** problems with automatic field generation that will prevent us from shipping it in 3.0

Comment (guardrex) [reply aspnetcore/blazor/components.md]: @rynowak How do you want to handle it in docs? Stick with our Pre7 guidance to code the backing fields? ... i.e., revert these backing field language/examples in this PR?

**Comment by guardrex** [comment]
I've reverted the backing field content. I've kept the `ElementRef` to `ElementReference` update.

I'm going to shoot this in for now. The PR resolves several other Pre8 content updates.

Let's fix up the backing field content when you can advise further.

**Pull request merged**